### ISSUE TITLE
LANLstar inner loop could be vectorized

### ISSUE THREAD

**Issue by jtniehof** [spacepy/spacepy#722]
The inner loop of _LANLcommon has a note that it could probably be replaced with aggregate/matrix ops with some care; this issue is just to capture that possibility.

**Comment by drsteve** [comment]
IIRC, the higher dimensions and broadcasting was difficult to get right (for me, at that time). The code may well be faster if that's done, but it's already very fast and easy to read/maintain. I'd suggest this is a very low priority, given that there's probably limited gain for whatever work is needed to actually make the change.

**Comment by jtniehof** [comment]
Agreed this is a low priority, only noted because I had to go into there for #723 and there was the node in the code, so capturing it here.
